frontend/__test__/SelectTest.test.js

```diff
@@ -0,0 +1,55 @@
+import { renderRouter, screen, fireEvent } from 'expo-router/testing-library';
+import Select from '../app/components/Select';
+
+it('renders the correct title', async () => {
+    const items = [
+        { label: 'Chile', value: 'Chile' },
+        { label: 'Peru', value: 'Peru' },
+    ];
+    const title = 'this is title';
+    const onChange = jest.fn();
+    
+
+    const MockComponent = jest.fn(() => <Select items={items} title={title} onChange={onChange} />);
+
+    renderRouter(
+        {
+            index: MockComponent,
+            'directory/a': MockComponent,
+            '(group)/b': MockComponent,
+        },
+        {
+            initialUrl: '/directory/a',
+        }
+    );
+    
+
+    expect(screen.getByDisplayValue('--- this is title ---')).toBeTruthy();
+});
+
+
+it ('renders the correct items', async () => {
```

Comment:
måske den kunne hedde 'onChange function is called when onchange is triggered'

Reply:
fordi den tester ikke om vi render de korrekte items (det kunne være nice med sådan en test men det ved jeg ikke om man kan)